build-swift-benchmarks-with-swiftpm.txt

```diff
@@ -0,0 +1,16 @@
+// Test that we can build the Swift benchmarks using SwiftPM.
+//
+// REQUIRES: have-swift-benchmarks
+//
+// RUN: rm -rf %t.dir
+// RUN: mkdir -p %t.dir/
+// RUN: cp -R %{swift_benchmarks_path} %t.dir/benchmark
```

Comment:
Wy copy? Why not just use the benchmark path? Other than that if everything works, LGTM.